src/libraries/Common/src/Interop/OSX/Interop.Libraries.cs

```diff
@@ -9,13 +9,13 @@ internal static partial class Libraries
         internal const string CoreFoundationLibrary = "/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation";
         internal const string CoreServicesLibrary = "/System/Library/Frameworks/CoreServices.framework/CoreServices";
         internal const string CFNetworkLibrary = "/System/Library/Frameworks/CFNetwork.framework/CFNetwork";
+        internal const string libobjc = "/usr/lib/libobjc.dylib";
         internal const string libproc = "libproc";
```

Comment:
Nit: You can do the same thing for libproc and fix https://github.com/dotnet/runtime/issues/24095 while you are on it.

Reply:
Good point. Might as well while I'm in here.